rllib/utils/torch_utils.py

```diff
@@ -233,66 +233,97 @@ def concat_multi_gpu_td_errors(
 
 @PublicAPI
 def convert_to_torch_tensor(
-    x: TensorStructType,
+    x,
     device: Optional[str] = None,
     pin_memory: bool = False,
+    use_stream: bool = False,
+    stream: Optional[Union["torch.cuda.Stream", "torch.cuda.classes.Stream"]] = None,
 ):
-    """Converts any struct to torch.Tensors.
+    """
+    Converts any (possibly nested) structure to torch.Tensors.
 
     Args:
-        x: Any (possibly nested) struct, the values in which will be
-            converted and returned as a new struct with all leaves converted
-            to torch tensors.
-        device: The device to create the tensor on.
-        pin_memory: If True, will call the `pin_memory()` method on the created tensors.
+        x: The input structure whose leaves will be converted.
+        device: The device to create the tensor on (e.g. "cuda:0" or "cpu").
+        pin_memory: If True, calls `pin_memory()` on the created tensors.
+        use_stream: If True, uses a separate CUDA stream for `Tensor.to()`.
```

Comment:
Can we explain here what happens in all these different cases? `use_stream=True and stream=None`, `use_stream=False (what is the default stream we are using then?)`, etc..